pkg/scheduler/algorithm/predicates/metadata_test.go

```diff
@@ -507,3 +507,235 @@ func TestPredicateMetadata_ShallowCopy(t *testing.T) {
 		t.Errorf("Copy is not equal to source!")
 	}
 }
+
+// TestGetPodsMatchingAffinity_AntiAffinity tests againt method getPodsMatchingAffinity
+// on Anti Affinity cases
+func TestGetPodsMatchingAffinity_AntiAffinity(t *testing.T) {
+	newPodAffinityTerms := func(keys []string) []v1.PodAffinityTerm {
+		var terms []v1.PodAffinityTerm
+		for _, key := range keys {
+			terms = append(terms, v1.PodAffinityTerm{
+				LabelSelector: &metav1.LabelSelector{
+					MatchExpressions: []metav1.LabelSelectorRequirement{
+						{
+							Key:      key,
+							Operator: metav1.LabelSelectorOpExists,
+						},
+					},
+				},
+				TopologyKey: "hostname",
+			})
+		}
+		return terms
+	}
+	tests := []struct {
+		name                 string
+		existingPods         []*v1.Pod
+		nodes                []*v1.Node
+		pod                  *v1.Pod
+		wantAffinityPods     map[string][]*v1.Pod
+		wantAntiAffinityPods map[string][]*v1.Pod
+		wantErr              bool
+	}{
+		{
+			name: "nil test",
+			nodes: []*v1.Node{
+				{ObjectMeta: metav1.ObjectMeta{Name: "nodeA", Labels: map[string]string{"hostname": "nodeA"}}},
+			},
+			pod: &v1.Pod{
+				ObjectMeta: metav1.ObjectMeta{Name: "aaa-normal", Labels: map[string]string{"aaa": ""}},
+			},
+			wantAffinityPods:     nil,
+			wantAntiAffinityPods: nil,
+		},
+		{
+			name: "non-affinity incoming pod causes a no-op", // the semmetric verification is covered elsewhere
```

Comment:
It seems this comment is unnecessary? As elsewhere seems to be test cases below?

Reply:
Removed the comment to avoid confusion.